### DISCUSSION TITLE
Command to enter your current risk on death gp into the public chat

### DISCUSSION THREAD

**Discussion by MangoMau** [runelite/runelite#18277] [Ideas]
Grap the value from the "items kept on death" interface.

For example !risk, to send a public chat message of how much you will lose if you die.
"playername risk: 2,340,00gp, with protect item on.

Include the "wilderness beyond level 20" death, as cosmetics and breakable items also have value to the killer.

**Comment by cdfisher** [comment]
It feels like this would largely be used for deathmatching which is against game rules, so I don't see this being something that would be added.

**Reply by MangoMau** [reply]
No, just PVP world pking actually.
The amount of time spent checking and stating your risk is crazy

**Reply by MangoMau** [reply]
Also, risking a lot of gp is not against the rules.
